### ISSUE TITLE
Deprecation warning

### ISSUE THREAD

**Issue by luisiam** [PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar#58]
I have a HW-Q990C, running HA 2024.1.2 with v1.0.2 of this integration. Start to get the following warning message in log

```
Entity media_player.hw_q990c (<class 'custom_components.smartthings_soundbar.media_player.SmartThingsSoundbarMediaPlayer'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <MediaPlayerEntityFeature.PAUSE|VOLUME_SET|VOLUME_MUTE|TURN_ON|TURN_OFF|VOLUME_STEP|SELECT_SOURCE|PLAY: 19853>, please create a bug report at https://github.com/PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar/issues and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation
```

**Comment by bernardesarthur** [comment]
Hi, I have a this same issue with Home Assistant Core 2024.03.

```
Registrador: homeassistant.helpers.entity
Fonte: helpers/entity.py:1597
Ocorreu pela primeira vez: 10:04:12 (1 ocorrências)
Último registro: 10:04:12

Entity media_player.home_theater (<class 'custom_components.smartthings_soundbar.media_player.SmartThingsSoundbarMediaPlayer'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <MediaPlayerEntityFeature.PAUSE|VOLUME_SET|VOLUME_MUTE|TURN_ON|TURN_OFF|VOLUME_STEP|SELECT_SOURCE|PLAY: 19853>, please create a bug report at https://github.com/PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar/issues and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation
```

Anyone can fix this?

**Comment by PiotrMachowski** [comment]
This doesn't cause any problem now, so it is not my priority. I might try to merge this integration to HA itself instead

**Comment by PiotrMachowski** [comment]
Fixed in `v1.0.3`
